fetch/api/response/response-consume.html

```diff
@@ -21,7 +29,7 @@
 
     function checkBodyBlob(response, expectedBody, checkContentType) {
       return response.blob().then(function(bodyAsBlob) {
-        if (checkContentType)
+        if (checkContentType !== false)
```

Comment:
This change is ok, but why should it be done? Is it the expected wpt coding style?